app/code/Magento/OfflineShipping/Model/ResourceModel/Carrier/Tablerate/RateQuery.php

```diff
@@ -51,7 +52,7 @@ public function prepareSelect(\Magento\Framework\DB\Select $select)
                 "dest_country_id = '0' AND dest_region_id = 0 AND dest_zip = '*'",
                 "dest_country_id = :country_id AND dest_region_id = 0 AND dest_zip = ''",
                 "dest_country_id = :country_id AND dest_region_id = 0 AND dest_zip = :postcode",
-                "dest_country_id = :country_id AND dest_region_id = 0 AND dest_zip = '*'"
```

Comment:
I am wondering why did you remove the line with the star?

Reply:
If you look at lines 49 and 54 in original code, you will see that there were two identical conditions in this OR statement. So we removed one.

Reply:
@magently yes, I missed that. You are right